tests/playwright/src/specs/container-smoke.spec.ts

```diff
@@ -154,7 +154,55 @@ describe('Verification of container creation workflow', async () => {
     await playExpect(startButton).toBeVisible();
   });
 
-  test('Deleting a container', async () => {
+  test(`Start a container from the Containers list`, async () => {
+    const navigationBar = new NavigationBar(page);
+    const containers = await navigationBar.openContainers();
+    const containersDetails = await containers.openContainersDetails(containerToRun);
+    await playExpect(containersDetails.heading).toBeVisible();
+    await playExpect(containersDetails.heading).toContainText(containerToRun);
+    // test state of container in summary tab
+    await navigationBar.openContainers();
+    await playExpect.poll(async () => await containers.containerExists(containerToRun)).toBeTruthy();
+
+    const containerRow = await containers.getContainerRowByName(containerToRun);
+    if (containerRow === undefined) {
+      throw Error(`Container: '${containerToRun}' does not exist`);
```

Comment:
We do not throw an error, we let the test to fail by using assertions. You should remove this part as you have already added `containerExists` asssertion...